themes/cardinal_service/lib/scss/opportunities/_opportunities.scss

```diff
@@ -1,17 +1,33 @@
 @charset 'UTF-8';
 
-.su-hero {
-  margin-bottom: 9.6rem;
+.opportunity {
```

Comment:
Not sure if this is the right class to target to make this styling specific for opportunity types only 🤔 Thoughts?

Reply:
I think that should be fine